main.js

```diff
@@ -0,0 +1,95 @@
+var addButton = document.querySelector('#add');
+var total = document.querySelector('#total-expenses');
+var expensesArr = [];
+var amount = document.querySelector('#expense-amount');
+var table = document.querySelector('table');
+var tableBody = document.querySelector('#tableBody');
+var amountClicked = false;
+var expenseClicked = false;
+
+
+var handleClick = function(event) {
+  var expenseName = document.querySelector('#expense-name');
+  var amount = document.querySelector('#expense-amount');
```

Comment:
You've already queried the DOM for these above, so no need to requery - they're cached above.